### PR TITLE
planet

### PR DESCRIPTION
**Pull request by sam-halfpenny**

### CODE REVIEW

File: planetbomber/planet.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<html>
+    <head>
+        <title>halfpenny gaming :) / BLOCK RUNNER</title>
+        <h1>BLOCK RUNNER (random)</h1>
```
Comment (will-0) [inline planetbomber/planet.html:4]: 👀👀👀

Comment (will-0) [reply planetbomber/planet.html:4]: Doesn't look like block runner to me :O

**Review by will-0 — CHANGES_REQUESTED** [review]
Sick!! A cool game man, where'd you get the idea for this one?

Other than the small change to title, it would also be great to have some instructions. This is just a suggestions so I'll leave it up to you whether you do it, but I think it would be cool the first time you see it :)

File: planetbomber/planet.html (hunk 1 above)
Comment (sam-halfpenny) [reply planetbomber/planet.html:4]: oh yeah

**Review by will-0 — APPROVED** [review]
Jazzy

**Pull request merged**